pages/index.js

```diff
@@ -1,7 +1,9 @@
+import PetList from "@/Component/PetList/PetList";
+
 export default function HomePage() {
   return (
     <div>
```

Comment:
You do not need this `div` element here as you are only returning one child here (`PetList`).